source/sass/type.scss

```diff
@@ -58,6 +58,11 @@ h6,
   text-transform: uppercase;
 }
 
+.p-heading {
```

Comment:
We should limit type styles to only the ones that are specified in the [style guide](https://foundation.mozilla.org/style-guide/).

- Page on production: https://foundation.mozilla.org/en/privacynotincluded/about/methodology
- Page on this review app: https://foundation-s-pni-update-p94fcd.herokuapp.com/en/privacynotincluded/about/methodology

@sabrinang can you suggest what existing text style works best for the third tier headings on the PNI's methodology page? (e.g., `Product Selection`, `Encryption`, `Security updates`, ... etc)